helm.go

```diff
@@ -82,16 +82,23 @@ func generate(log Logger, si ServiceItem, basePath, outputDir string) error {
 	return nil
 }
 
-func helmInstall(log Logger, si ServiceItem, image, tag, outputDir string) (string, error) {
+func helmInstall(log Logger, si ServiceItem, service Service, outputDir string, _ bool) (string, error) {
 	log.Printf("Running helm install for service: %v", si.Name)
 	chartPath := path.Join(outputDir, si.Name)
 	cmd := []string{
-		"helm", "install", si.Name, chartPath,
+		"helm", "upgrade", si.Name, chartPath,
 		"-f", path.Join(chartPath, defaultOverridesFile),
 		"-n", si.namespace,
+		"--install",
 		"--set", fmt.Sprintf("group=%v", si.group),
-		"--set", fmt.Sprintf("image.repository=%v:%v", image, tag),
+		"--set", fmt.Sprintf("image.repository=%s", service.Image),
+		"--set", fmt.Sprintf("image.tag=%s", service.Tag),
+		"--set", fmt.Sprintf("image.tag=%s", service.Tag),
+		"--set", fmt.Sprintf("metadata.name=%s", service.Name),
+		"--set", fmt.Sprintf("metadata.component=%s", si.group),
+		"--set", fmt.Sprintf("metadata.namespace=%s", si.namespace),
```

Comment:
We should map these to example chart yamls (example: were using `app.kubernetes.io/part-of: {{ .Values.group }}` label at the moment)

Reply:
Afaik charts are using that in the catalogue playground. Perhaps a better way to is to have a global configuration which will be injected.

Reply:
Agreed. Perhaps we can leverage helm subcharts for this.